backend/notifications/signals/handlers.py

```diff
@@ -46,3 +46,28 @@ def get_data_for_websocket(sender, instance, **kwargs):
             }
         }
     )
+
+
+@receiver(pre_delete, sender=Notification)
```

Comment:
```suggestion
@receiver(post_delete, sender=Notification)
```
Давай оставим post delete.
Все таки обращения по индексу здеь нет как и нет отправки почты.
А при пре делет смысл отправки обновлений в вебсокет теряется, ведь количество уведомлений еще не изменилось, тк объект не удален